pageserver/src/http/openapi_spec.yml

```diff
@@ -212,6 +212,12 @@ paths:
               schema:
                 type: string
                 format: date-time
+        "412":
+          description: No timestamp is found for given LSN, e.g. if there had been no commits till LSN
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PreconditionFailedError"
```

Comment:
1. PR should link the corresponding cloud.git change
2. Have you checked how the cloud.git code will deal with the changed status code? Worth mentioning in the PR description